content/backmatter.tex

```diff
@@ -357,6 +357,9 @@ \section{Version 1.5}
 
 \item The \VAR{SHMEM\_SYNC\_SIZE} constant was added.
 \\See Section \ref{subsec:library_constants}.
+%
+\item \newtext{Expanded the type support for RMA and AMO operations.
+\\ See Sections \ref{sec:rma} and \ref{sec:amo}.}
```

Comment:
I think it would be a good idea to list all of the types that were added in the changelog entry.  In the future, if a user is trying to figure out whether their OpenSHMEM library should support a given type, this will be useful information.